content/docs/introduction/comparison.md

```diff
@@ -246,31 +246,30 @@ environment, then Prometheus is a good choice.
 
 ## Prometheus vs. Sensu
 
-[Sensu](https://sensu.io) is a composable monitoring pipeline that can reuse existing Nagios checks.
+[Sensu](https://sensu.io) is an open source monitoring and observabliity pipeline with a commercial distribution. It can reuse existing Nagios plugins for compatibility.
```

Comment:
just monitoring pipeline is enough.

The "for compatibility" doesn't add anything.